src/Bridge/Doctrine/Orm/SubresourceDataProvider.php

```diff
@@ -142,7 +142,9 @@ private function buildQuery(array $identifiers, array $context, QueryNameGenerat
         $classMetadata = $manager->getClassMetadata($identifierResourceClass);
 
         if (!$classMetadata instanceof ClassMetadataInfo) {
-            throw new RuntimeException("The class metadata for $identifierResourceClass must be an instance of ClassMetadataInfo.");
+            throw new RuntimeException(
+                "The class metadata for $identifierResourceClass must be an instance of ClassMetadataInfo."
```

Comment:
Code style change? I'd revert it.

Reply:
Reverted. I guess it sneaked out because PHPStorm was too enthusiastic.